examples/step-61/doc/results.dox

```diff
@@ -1,20 +1,25 @@
 <h1>Results</h1>
 
-We run the test example $p = \sin(\pi x) \sin(\pi y)$ with homogenous Dirichelet 
-boundary conditions in the domain $\Omega = (0,1)^2$. And  $\mathbf{K}$ is the 
-identity matrix. We test it on $\mbox{WG}(Q_0,Q_0;RT_{[0]})$, $\mbox{WG}(Q_1,Q_1;RT_{[1]})$ 
-and $\mbox{WG}(Q_2,Q_2;RT_{[2]})$. We will visualize pressure values in interiors 
-and on faces. We want to see the pressure maximum is around 1 and the minimum 
-is around 0. With the mesh refinement, the convergence rates of pressure, 
-velocity and flux should be around 1 on $\mbox{WG}(Q_0,Q_0;RT_{[0]})$ , 2 on 
-$\mbox{WG}(Q_1,Q_1;RT_{[1]})$, and 3 on $\mbox{WG}(Q_2,Q_2;RT_{[2]})$.
+We run the program with a right hand side that will produce the solution
+$p = \sin(\pi x) \sin(\pi y)$ and with homogenous Dirichlet 
```

Comment:
```suggestion
$p = \sin(\pi x) \sin(\pi y)$ and with homogeneous Dirichlet 
```
(I only noticed this because of #7979)

Reply:
Ha -- I think that both spellings are correct. But I'm happy to go with the generally used one.